### PR TITLE
[fix] Change Gap between the FAB buttons.

### PR DESCRIPTION
**Pull request by wndudqus**
## Description

style fix.
before:
the gap between the FAB buttons are not the same across the environment.

after
1. change the unit for the gap. % => px
2. gap will be decided with the height of the FAB view

I checked IOS, Web, and Android environments.
Check below.

(ios)
https://user-images.githubusercontent.com/35381940/135742330-ce241049-4f54-42d1-a4dc-1b2444e63ad9.mov

(web)
https://user-images.githubusercontent.com/35381940/135742409-2073bf20-fe78-411a-801f-d6e6f0d29f9e.mov

(android)
https://user-images.githubusercontent.com/35381940/135743099-84e53125-93ff-4368-98d6-5013297dc733.mov

## Test Plan
check the demo. 

## Related Issues
[related Issue](https://github.com/dooboolab/dooboo-ui/issues/135#issue-1014072268)

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test:all` and make sure nothing fails. You can run `yarn test -u` to update snapshots if needed.
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by wndudqus** [comment]
@yujong-lee @hyochan 
I created PR for [related Issue](https://github.com/dooboolab/dooboo-ui/issues/135#issue-1014072268). 
thanks.

File: main/FAB/index.tsx [hunk 1]
```diff
@@ -46,10 +53,12 @@ function FloatingActionButtons<Item extends FABItem = FABItem>({
...
   theme: DoobooTheme;
 }): ReactElement {
+  const GAP = 15;
   const {FAB, FABItem, buttonSize = 'large', iconSize = 24} = styles ?? {};
 
-  const spinValue = useRef(new Animated.Value(0));
-  const positionValue = useRef(new Animated.Value(0));
+  const spinValue = useRef<Animated.Value>(new Animated.Value(0));
+  const positionValue = useRef<Animated.Value>(new Animated.Value(0));
+  const FABHeight = useRef<number>(0);
```
Comment (yujong-lee) [inline main/FAB/index.tsx:61]: I suggest you to write like this.

```ts
const spinValue = useRef(new Animated.Value(0));
const positionValue = useRef(new Animated.Value(0));
const FABHeight = useRef(0);
```

See [this](https://github.com/dooboolab/dooboo-ui/pull/110#discussion_r706622897) for details.

File: main/FAB/index.tsx [hunk 2]
```diff
@@ -46,10 +53,12 @@ function FloatingActionButtons<Item extends FABItem = FABItem>({
 }: FABProps<Item> & {
   theme: DoobooTheme;
 }): ReactElement {
+  const GAP = 15;
```
Comment (yujong-lee) [inline main/FAB/index.tsx:56]: How about moving this to `styles`?

**Review by yujong-lee — CHANGES_REQUESTED** [review]
Great work! Just few things I can suggest for you.

File: main/FAB/index.tsx (hunk 1 above)
Comment (wndudqus) [reply main/FAB/index.tsx:61]: oh, I see. 
thank you for pointing it out!

File: main/FAB/index.tsx (hunk 2 above)
Comment (wndudqus) [reply main/FAB/index.tsx:56]: That is a good idea thanks!

File: main/FAB/index.tsx (hunk 1 above)
Comment (wndudqus) [reply main/FAB/index.tsx]: @yujong-lee 
I reflected this review thanks.

File: main/FAB/index.tsx (hunk 2 above)
Comment (wndudqus) [reply main/FAB/index.tsx]: @yujong-lee 
I reflected this review thanks.

**Review by hyochan — APPROVED** [review]
LGTM!

**Review by yujong-lee — APPROVED** [review]
👍

**Pull request merged**